src/libraries/System.Reflection/tests/ModuleTests.cs

```diff
@@ -82,61 +82,21 @@ public void GetTypeTest(string className, Type expectedType)
         [Fact]
         public void FilterTypeName_DelegateFiltersExpectedTypes()
         {
-            Assert.NotNull(Module.FilterTypeName);
-            Assert.Same(Module.FilterTypeName, Module.FilterTypeName);
-            Assert.NotSame(Module.FilterTypeName, Module.FilterTypeNameIgnoreCase);
```

Comment:
Were some tests removed? I can find the first two tests elsewhere but not the third:
```Assert.NotSame(Module.FilterTypeName, Module.FilterTypeNameIgnoreCase);```

Reply:
Added back